etsd/keys/templates/keys/publickey_detail.html

```diff
@@ -20,6 +44,10 @@
       <div class="alert alert-warning">
         The public key needs confirmation by the administration in order to be used. 
       </div>
+      {% if perms.core.admin %}
+        <!-- Button trigger modal -->
+      <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#confirmModal">{% translate "Confirmation" %}</button>
```

Comment:
I prefer to have two buttons: Accept in green and Reject in red. Both will display the confirmation and if the user agrees they will submit the form properly.